bool reset
front_end/stereoLandmarks latestFrame
---
front_end/windowStatus state 
